.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,18 @@
 ## PR Checklist
 Please check if your PR fulfills the following requirements:
 
+- [ ] The commit subject follows the [Conventional Commits spec](https://github.com/zeke/semantic-pull-requests)
+- [ ] The commit message follows the [EdgeX Contributor Guide](https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide)
```

Comment:
I thought we are dropping these since the commit bot is enforcing it.

Reply:
Sure - I removed the types check off, but left these as I don't think the bot will make sure we adhere to the optional and recommended parts of the spec nor our own guidelines.  But happy to pull these out too.  Makes for a faster PR process.